Evaluation/Input/Content/Section2.2_Data.md

```diff
@@ -25,7 +25,6 @@ A literature search was performed to collect available clinical data on fluvoxam
 | [Carillo 1996](#5-References)<sup>+</sup> | p.o. | 50 | HV | - | - | m/f | 5 | e.c. Tab | Non-smoking PM |
 | [Spigset 1997](#5-References)<sup>+</sup> | p.o.      | 50                              | HV           | -                    | -                      |         | 5     | e.c. Tab  | PM                                |
 | [Spigset 1997](#5-References) | p.o.      | 50                              | HV           | -                    | -                      |         | 5     | e.c. Tab  | EM                                |
-| [DeVries 1992](#5-References) | p.o.      | 50 b.i.d.                       | HV + Elderly | -                    | -                      | m/f     | 25    | e.c. Tab  | -                                 |
```

Comment:
In the References.md **DeVries 1992** was replaced by **DeVries 1993**, but here **DeVries 1992** was just removed without adding a new entry?

Reply:
Not replaced, 1992 has been removed (as is not used), 1993 has been there before, see line 33